librz/include/rz_util/rz_sys.h

```diff
@@ -79,6 +79,7 @@ RZ_API int rz_sys_system(const char *command);
 #else
 #define rz_sys_system system
 #endif
+#define rz_sys_system_(cmd) RZ_V_NOT (rz_sys_system (cmd), -1)
```

Comment:
what is this? And why is it necessary? I'm not a fan of this `_` suffix.

Reply:
> what is this?

`RZ_V_NOT` is defined as

```c
#define RZ_V_NOT(op, fail_ret) \
	if ((op) == (fail_ret)) \
		eprintf (#op" at %s:%d failed: %s\n", __FILE__, __LINE__, strerror (errno))
```

i.e. it checks whether the operation fails and prints an error message to stderr if so ... `rz_sys_system_` is its specialization to `rz_sys_system`.

----

> And why is it necessary?

Because of the following warning (https://github.com/rizinorg/rizin/pull/260/checks?check_run_id=1708037157#step:12:904):

![Wunused-result](https://user-images.githubusercontent.com/12002672/104816460-5156b200-5856-11eb-95f2-4b9c5725d226.PNG)

and because we treat warnings as errors.

----

> I'm not a fan of this `_` suffix.

It's the most lightweight naming that I could think of. Naming it `rz_sys_system_checked` / `write_checked` / `read_checked` etc. is possible but really it does appear that the return value isn't of much importance and this is done just so the compiler is pleased and there's some minimum error handling.

Reply:
By "return value isn't of much importance" I mean the call is unlikely to return `-1` ... 879d9692460bb6a8ecd87d312b4e4971e1934b32 does at least print an error message if that happens instead of staying silent.

Reply:
What about `x` prefix/suffix? I remember of some codebase that use things like `xmalloc` that does malloc+exit if fail. In theory, though, we should check the return value and act accordingly.

Reply:
> What about `x` prefix ...?

Ok lgtm dee1b9a6b0c2bc7eea3a3d75b8a95d41ea54ccc7.

> In theory, though, we should check the return value and act accordingly.

I agree but that's too much for this pr.